samples/snippets/translate_v3beta1_translate_document.py

```diff
@@ -41,7 +41,11 @@ def translate_document(project_id: str, file_path: str):
         }
     )
 
-    # To view translated document, write `response.document_translation.byte_stream_outputs` to file.
+    # To view translated document, uncomment the code below.
```

Comment:
nit: `view` can be slightly misleading since the code snippet outputs the content onto the file but not actually letting the user see the content until they open it.